### DISCUSSION TITLE
Add TouchEffect

### DISCUSSION THREAD

**Discussion by SimonSchnitker** [CommunityToolkit/Maui#611] [New Feature Discussions]
Hello, everyone,

I was about to add a TouchEffect when I realized it wasn't there. In the Xamarin.Forms CommunityToolkit, the following properties could be assigned to the layouts in order to create a native TouchEffect on the layout:

```
xmlns:xct="http://xamarin.com/schemas/2020/toolkit"

<StackLayout xct:TouchEffect.NativeAnimation="True"
             xct:TouchEffect.NativeAnimationColor="Gray"
             xct:TouchEffect.Command="{Binding AnyCommand, Mode=OneWay}">
```

Will this feature be added in the future?

**Comment by VladislavAntonyuk** [comment]
We have an issue which you can track. 
if you need it immediately you can use MauiCompat package which has all features available in Xamarin Community Toolkit

**Comment by CliffAgius** [comment]
Closed as answered
